docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -31,6 +31,15 @@ isTocNested: true
 ## Overview
 Follow the steps below to run the open-source [oltpbench](https://github.com/oltpbenchmark/oltpbench) TPC-C workload against YugabyteDB YSQL. [TPC-C](http://www.tpc.org/tpcc/) is a popular online transaction processing benchmark that provides metrics you can use to evaluate the performance of YugabyteDB for concurrent transactions of different types and complexity that are either either executed online or queued for deferred execution.
 
+### Results at a glance
+| Warehouses| TPMC | Efficiency (approx) | Cluster Details
+-------------|-----------|------------|------------|
+10   | 126   | 97.97% | 3 nodes of type `c5.large` (2 vCPUs)
```

Comment:
Any reason why we didn't use c5d.large here instead of c5.large?

Reply:
We wanted to use a 2vCPU machine to indicate that the workload can run on small machines.

Reply:
c5d.large is also 2 vCPU, right? We decided to move away from c5.xx and use c5d.xx.